RotaryWheelControl/RotaryWheel.xaml.cs

```diff
@@ -143,6 +145,52 @@ public RotaryWheel()
             };
         }
 
+        private void SpinTo(int itemIndex, int durationInSec = -1)
+        {
+            Random r = new Random(DateTime.Now.Millisecond);
+            var angleFromYAxis = 360 - Angle;
+            SelectedItem = _pieSlices
+                .SingleOrDefault(p => p.StartAngle <= angleFromYAxis && (p.StartAngle + p.Angle) > angleFromYAxis);
+
+            int count = _pieSlices.Count;
+            int currIndex = _pieSlices.IndexOf(SelectedItem);
+            int fullSpin = itemIndex / count;
+            itemIndex = itemIndex % count;
+            int steps = currIndex-itemIndex;
+            if (steps < 0)
+                steps = count + steps;
+
+            var startAngle = SelectedItem.StartAngle + SelectedItem.Angle / 2;
```

Comment:
Shouldn't `SelectedItem.StartAngle` be sufficient? Why is the `.Angle/2` necessary? I'm a couple years removed from this codebase.